amp.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * Plugin Name: AMP
- * Description: An easier path to great Page Experience for everyone. Powered by AMP.
+ * Description: An easier path to great Page Experience for everyone. Powered by AMP. <em class="amp-deletion-notice"><strong>Uninstall Note:</strong> To control whether all data from this plugin is deleted at uninstallation, first activate the plugin, go to the Other section on the Settings screen, and set the “Delete plugin data at uninstall” toggle.</em>
```

Comment:
Wondering if the install note should be on a separate line for better visibility 🤔

```suggestion
 * Description: An easier path to great Page Experience for everyone. Powered by AMP. <br><em class="amp-deletion-notice"><strong>Uninstall Note:</strong> To control whether all data from this plugin is deleted at uninstallation, first activate the plugin, go to the Other section on the Settings screen, and set the “Delete plugin data at uninstall” toggle.</em>
```

Before | After
---|---
<img width="833" alt="image" src="https://user-images.githubusercontent.com/16200219/136283611-d65f27c8-9f0e-4fe8-aec8-facad3efa59a.png"> | <img width="795" alt="image" src="https://user-images.githubusercontent.com/16200219/136283574-a2e3d7d9-c5e8-4653-b1be-b433c300e1e6.png">

Reply:
Ideally, yes. Unfortunately, the `br` tag is not allowed. Only `abbr`, `acronym`, `code`, `em`, `strong`, and `a` are allowed: https://github.com/WordPress/wordpress-develop/blob/ed9f437fc098ba2c9ba18eda0fa341ecd0476ca9/src/wp-admin/includes/plugin.php#L173-L186

Reply:
Oh I see, unfortunate indeed.